asset_data.py

```diff
@@ -9,31 +9,44 @@ class DataManager:
     def __init__(self):
 
         self.cg_endpoint = "https://api.coingecko.com/api/v3"
-        self.cg_asset_data = {}
+        self.cg_assets = "bitcoin,ethereum,solana,dogecoin"

```

Comment:
Can delete this line, was used for old API call (/simple/price), just need list for new API cal